### PR TITLE
Set TLS cipher suites for apiserver

### PR DESCRIPTION
**Pull request by stone-z**
Towards https://github.com/giantswarm/giantswarm/issues/4812

Allows the following TLS ciphers:
```
TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256
TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384
TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305
TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256
TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384
TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305
TLS_RSA_WITH_AES_256_GCM_SHA384
TLS_RSA_WITH_AES_128_GCM_SHA256
```

Based on recommendations from @pipo02mix, [SSLLabs](https://github.com/ssllabs/research/wiki/SSL-and-TLS-Deployment-Best-Practices) and [rancher](https://rancher.com/docs/rancher/v2.x/en/installation/options/tls-settings/).

## Checklist

- [x] Update changelog in CHANGELOG.md.

### CODE REVIEW

File: files/manifests/k8s-api-server.yaml [hunk 1]
```diff
@@ -53,6 +53,7 @@ spec:
     - --logtostderr=true
     - --tls-cert-file=/etc/kubernetes/ssl/apiserver-crt.pem
     - --tls-private-key-file=/etc/kubernetes/ssl/apiserver-key.pem
+    - --tls-cipher-suites=TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384,TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305,TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384,TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305,TLS_RSA_WITH_AES_256_GCM_SHA384,TLS_RSA_WITH_AES_128_GCM_SHA256
```
Comment (MarcelMue) [inline files/manifests/k8s-api-server.yaml:56]: This needs to be tested - I am uncertain if the apiserver will be happy with this way of passing them in.

Comment (stone-z) [reply files/manifests/k8s-api-server.yaml]: I believe you are correct, updated to use the kubelet config files. Still have to try it out

Comment (stone-z) [reply files/manifests/k8s-api-server.yaml]: Actually, it looks like this method is deprecated for the kubelet but not for apiserver. So I think I need both
https://kubernetes.io/docs/reference/command-line-tools-reference/kube-apiserver/

**Comment by stone-z** [comment]
Tested by Fer on `geckon` https://gigantic.slack.com/archives/CHAHY6VMF/p1590652277159200

**Pull request merged**